lib/Templating/Twig/TwigThemeLoader.php

```diff
@@ -94,25 +85,18 @@ public function getNamespaces()
         return $this->innerFilesystemLoader->getNamespaces();
     }
 
-    public function setPaths($paths, $namespace = self::MAIN_NAMESPACE)
+    public function setPaths($paths, $namespace = FilesystemLoader::MAIN_NAMESPACE)
     {
-        parent::setPaths($paths, $namespace);
         $this->innerFilesystemLoader->setPaths($paths, $namespace);
     }
 
-    public function addPath($path, $namespace = self::MAIN_NAMESPACE)
+    public function addPath($path, $namespace = FilesystemLoader::MAIN_NAMESPACE)
     {
-        parent::addPath($path, $namespace);
         $this->innerFilesystemLoader->addPath($path, $namespace);
     }
 
-    public function prependPath($path, $namespace = self::MAIN_NAMESPACE)
+    public function prependPath($path, $namespace = FilesystemLoader::MAIN_NAMESPACE)
     {
         $this->innerFilesystemLoader->prependPath($path, $namespace);
     }
-
-    public function findTemplate($template, $throw = true)
-    {
-        return parent::findTemplate($template, $throw);
-    }
```

Comment:
I'm probably missing something, but what change made this removal possible?

Reply:
Sorry for late reply @lolautruche. This method became protected in parent class after migrating to Twig 2.11.